chapter5/code-for-math/test_empty_problems.py

```
@@ -0,0 +1,54 @@
+from pathlib import Path
+from types import SimpleNamespace
+
+import demo as cfm
+
+
+class _Resp:
+    choices = [
+        SimpleNamespace(
+            message=SimpleNamespace(content="FINAL ANSWER: 1", tool_calls=None)
+        )
+    ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)